File: luci-app-honk/luasrc/model/cbi/honk/node.lua
local honk = require "luci.model.honk_tools"
local m, s

m = Map("honk", translate("Node Settings"), translate("Configure nodes and groups for HONK."))

local node_file = "/etc/honk/config.d/node.dae"

s = honk.init_editor(m, "node")
honk.add_editor(s, node_file, "nodeconf", translate("Node Configuration"))

return m

File: luci-app-honk/root/www/luci-static/resources/view/honk/node.js
'use strict';
'require honk.common as honk';

return honk.createConfigFileView(
	'/etc/honk/config.d/node.dae',
	_('Node Settings'),
	_('Configure nodes and groups for HONK.'),
	_('Node Configuration'),
	_('Node configuration saved and service reloaded.')
);
